### PR TITLE
Fix for MySQL 5.5 FK order bug

### PR DESCRIPTION
**Pull request by thinQ-skeema**
Fix for [issue#83](https://github.com/skeema/skeema/issues/83)

### CODE REVIEW

File: fs/dir.go [hunk 1]
```diff
@@ -428,10 +428,7 @@ func (dir *Dir) InstanceDefaultParams() (string, error) {
 	v.Set("default_storage_engine", "'InnoDB'")
 	v.Set("sql_quote_show_create", "1")
 
-	flavorFromConfig := tengo.NewFlavor(dir.Config.Get("flavor"))
-	if flavorFromConfig.HasDataDictionary() {
-		v.Set("information_schema_stats_expiry", "0")
-	}
+	v.Set("information_schema_stats_expiry", "0")
```
Comment (evanelias) [inline fs/dir.go]: These changes shouldn't be here... this is why the test is failing.

**Comment by evanelias** [comment]
EDIT: sorry that should be something like `upstream/master` rather than `origin/master`. You may need to [manually configure the upstream remote](https://help.github.com/en/articles/configuring-a-remote-for-a-fork) first though.

**Comment by thinQ-skeema** [comment]
I fixed the fs/dir.go first - to pass the tests. Then I verified that nothing else remained:
git reset --hard upstream/master
HEAD is now at 2f9c848 lint: reformat files prior to problem checks, not after
git push --force
Total 0 (delta 0), reused 0 (delta 0)
To https://github.com/thinQ-skeema/skeema.git
 + 5abf3fa...2f9c848 master -> master (forced update)

**Comment by evanelias** [comment]
The PR appears blank and closed now?

The idea was to reset to upstream/master to get rid of the old commits, and *then* re-run the `dep` step and re-commit, in order to get that in place as a single commit.

Would you like to re-run the `dep` step from https://github.com/skeema/skeema/issues/83#issuecomment-511085095 and re-open the PR? Or would you prefer for me to do these steps on my end? Thanks. Hoping to get this change in place for the next binary release, which I'm aiming for either tomorrow or Tues.

**Comment by thinQ-skeema** [comment]
I was unable to re-open this closed PR and created the new PR after re-running the `dep` step.

**Pull request closed without merging**